### ISSUE TITLE
Ability to keep location fixed when zooming in (with option to still move the center point)

### ISSUE THREAD

**Issue by carrieseltzer** [inaturalist/INaturalistIOS#160]
When I edit the location of an observation on my iPhone/iPad, I would like there to be a way to keep the center of the circle in precisely the same point as I zoom in. As it is, it's hard to keep the center in the same place while zooming so I worry about losing the original location when I'm trying to zoom in to make it more precise. 

There should obviously also be a way to move the center point. Not sure what the UI could be like to switch between those options.

Not high priority, but I wanted to get this down while I was thinking of it.

**Comment by alexshepard** [comment]
:thumbsup:

**Comment by alexshepard** [comment]
@carrieseltzer are you still thinking about this in the app? would something like zoom buttons [+] and [-] work for you?
@joellebel what do you think about zoom buttons on the edit location map as a way to support this without redesigning?

**Comment by joellebel** [comment]
Sure. Will whip up a quick design.

**Comment by carrieseltzer** [comment]
Yeah, I still wish for something like this if I'm adding observations after the fact rather than using hte app to take photos in the field. Zoom buttons could work. Thanks!

**Comment by joellebel** [comment]
@alexshepard Got an idear... Can you let me know a few days before you would like to begin this and I will design it. Gracias!
